savas_wordlist.txt

```diff
@@ -375,3 +375,16 @@ Kramdown
 backtick
 backticks
 Dockerized
+DrupalCon
+Dries
+Dries's
+Buytaert
+iframe
+APIs
+WordPress
+WordPress's
+interconnectedness
+architecting
+rearchitecture
+Driesnote
+Drupalcons
```

Comment:
@chrisarusso We need to replace "Drupalcons" with "DrupalCons" to ensure proper capitalization in the future, and it looks like "re-architecture" needs a hyphen